backend/danswer/db/embedding_model.py

```diff
@@ -48,14 +50,14 @@ def create_embedding_model(
     return embedding_model
 
 
-def get_model_id_from_name(
-    db_session: Session, embedding_provider_name: str
-) -> int | None:
+def get_embedding_provider_from_provider(
```

Comment:
mmm a bit confusing in terms of the name, makes sense when you read the code but ideally it can be clearer?